src/vs/workbench/api/electron-browser/mainThreadTask.ts

```diff
@@ -485,6 +515,24 @@ export class MainThreadTask implements MainThreadTaskShape {
 		});
 	}
 
+	public $extensionCallbackTaskComplete(id: string): Promise<void> {
+		return new Promise<void>((resolve, reject) => {
+			this._taskService.getActiveTasks().then((tasks) => {
+				for (let task of tasks) {
+					if (id === task._id) {
+						this._taskService.extensionCallbackTaskComplete(task).then((value) => {
+							resolve(undefined);
+						}, (error) => {
```

Comment:
If we're just going to eat the error here it probably isn't worth having this here at all. We can either do something with the error (log, alert, reject), or just delete the error block.